docs/en_US/Tutorial/TuningSystems.md

```diff
@@ -0,0 +1,8 @@
+# Automatically tune systems with NNI
+
+As computer systems and networking get increasingly complicated, optimizing them manually with explicit rules and heuristics becomes harder than ever before, sometimes impossible. At Microsoft Research Asia, our AutoSys project applies learning techniques provided by NNI to large-scale system performance tuning. Based on AutoSys, we have built a tool to help many crucial system scenarios within Microsoft. These scenarios include multimedia search for Bing (e.g., tail latency reduced by up to ~40%, and capacity increased by up to ~30%), job scheduling for Bing Ads (e.g., tail latency reduced by up to ~13%), and so on.
```

Comment:
Due the nni is an opensource project,  I think we don't need to Mention "MSR...Autosys project"...

You could delete it "At Microsoft Research Asia, our AutoSys project applies learning techniques provided by NNI to large-scale system performance tuning."

Also delete " These scenarios include multimedia search for Bing (e.g., tail latency reduced by up to ~40%, and capacity increased by up to ~30%), job scheduling for Bing Ads (e.g., tail latency reduced by up to ~13%), and so on"